python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2509,9 +2515,7 @@ def _parse_param(self, key, value, name, shape):
 
             array_ndim = len(np_array.shape)
             if array_ndim == 0:
-                new_array = np.empty([1], dtype=np_array.dtype)
```

Comment:
Removed this because we want to pass the scalar as scalar only and not as a tensor of rank 1.